<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>565</width>
    <height>320</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Lincom</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{background:rgb(229, 255, 239)}
QMenuBar{background:url(:/new/prefix2/src/header.bmp)}
QStatusBar{background:url(:/new/prefix2/src/header.bmp)}
QPushButton{background:url(:/new/prefix2/src/header.bmp)}
QLCDNumber{background:url(:/new/prefix2/src/lcd.bmp)}
QLabel{background:url(:/new/prefix2/src/header.bmp)}







</string>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>portset</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>70</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>bound:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>115</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>databits:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>155</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>parity:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>197</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>stopbits:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>36</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>port:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>244</y>
       <width>63</width>
       <height>17</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>ms:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="portNameComboBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>84</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ttymxc1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttymxc2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttymxc3</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="baudRateComboBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>84</width>
       <height>27</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>12</number>
     </property>
     <item>
      <property name="text">
       <string>50</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>75</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>110</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>134</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>150</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>300</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>56000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>76800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>128000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>256000</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="dataBitsComboBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>84</width>
       <height>27</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="parityComboBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>84</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>N</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>O</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>E</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="stopBitsComboBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>84</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
    <widget class="QSpinBox" name="delayspinBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>237</y>
       <width>84</width>
       <height>27</height>
      </rect>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>5000</number>
     </property>
     <property name="value">
      <number>200</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>371</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>receive</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>28</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <widget class="QRadioButton" name="ccradioButton">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>13</y>
        <width>58</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>以字符的方式发送和接收数据</string>
      </property>
      <property name="text">
       <string>char</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="chradioButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>13</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>以字符方式发送, 以十六进制方式显示</string>
      </property>
      <property name="text">
       <string>HEX</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="dcradioButton">
      <property name="geometry">
       <rect>
        <x>135</x>
        <y>13</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>以字符方式发送, 以十进制方式显示</string>
      </property>
      <property name="text">
       <string>DEC</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="recvbyteslcdNumber">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>12</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>已接收字节数</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QPushButton" name="clearUpBtn">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>clear</string>
      </property>
     </widget>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>70</y>
       <width>361</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="sendMsgLineEdit">
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>250</y>
      <width>291</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>10000</number>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="obocheckBox">
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>217</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>continue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendmsgBtn">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>250</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>send</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>220</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>send:</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>55</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionClose"/>
   <addaction name="separator"/>
   <addaction name="actionAdd"/>
   <addaction name="actionReadPortInterval"/>
   <addaction name="separator"/>
   <addaction name="actionClearBytes"/>
   <addaction name="actionCleanPort"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionAdd">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>addport</string>
   </property>
   <property name="toolTip">
    <string>addport</string>
   </property>
   <property name="statusTip">
    <string>addport</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>openport</string>
   </property>
   <property name="statusTip">
    <string>open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>closeport</string>
   </property>
   <property name="statusTip">
    <string>closeport</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>:/new/prefix1/src/exit.png</normaloff>:/new/prefix1/src/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出程序</string>
   </property>
   <property name="statusTip">
    <string>退出程序</string>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset>
     <normaloff>:/new/prefix1/src/mesage.png</normaloff>:/new/prefix1/src/mesage.png</iconset>
   </property>
   <property name="text">
    <string>关于Lincom</string>
   </property>
   <property name="toolTip">
    <string>关于Lincom</string>
   </property>
   <property name="statusTip">
    <string>关于Lincom</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>savedata</string>
   </property>
   <property name="toolTip">
    <string>savedata</string>
   </property>
   <property name="statusTip">
    <string>savedata</string>
   </property>
  </action>
  <action name="actionClearBytes">
   <property name="text">
    <string>clear</string>
   </property>
   <property name="toolTip">
    <string>clear</string>
   </property>
   <property name="statusTip">
    <string>clear</string>
   </property>
  </action>
  <action name="actionReadPortInterval">
   <property name="text">
    <string>readms</string>
   </property>
   <property name="toolTip">
    <string>read ms</string>
   </property>
   <property name="statusTip">
    <string>read ms</string>
   </property>
  </action>
  <action name="actionLoadfile">
   <property name="text">
    <string>readfile</string>
   </property>
   <property name="toolTip">
    <string>read file</string>
   </property>
   <property name="statusTip">
    <string>read file</string>
   </property>
  </action>
  <action name="actionCleanPort">
   <property name="text">
    <string>clearport</string>
   </property>
   <property name="toolTip">
    <string>clearport</string>
   </property>
   <property name="statusTip">
    <string>clearport</string>
   </property>
  </action>
  <action name="actionWriteToFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>writefile</string>
   </property>
   <property name="toolTip">
    <string>writefile</string>
   </property>
   <property name="statusTip">
    <string>writefile</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
